### DISCUSSION TITLE
:Git produces empty split (no content)

### DISCUSSION THREAD

**Discussion by mellery451** [tpope/vim-fugitive#2089] [Q&A] [answered]
I'm not getting typical fugitive status window from :Git as of a day or two ago. This has worked fine forever until now. Is this command deprecated, or perhaps something else on my system might have changed (git version perhaps?). Any other tips to help diagnose?

**Comment by tpope** [comment]
I wouldn't expect a Git upgrade to cause a quiet failure like that. The Fugitive version, on the other hand, might. In particular, 9a13fc87c4ab83ea81e71ccad9b846a5eb31a998 recently changed the filename of the status buffer, so you might try its parent commit and see if that changes things.

**Comment by mellery451** [comment]
resetting to 23b321b seems to have fixed it. I update plugins pretty regularly, but maybe this commit actually just merged recently. in any event, is this a bug or something I should workaround another way on HEAD?

**Reply by tpope** [reply]
Possibly a bug, more likely an issue on your end. Make sure you have just 1 copy of Fugitive installed. Check `:scriptnames` and make sure all files named `fugitive.vim` come from the same place.

**Comment by mellery451** [comment, marked as answer]
Turns out I had a non-empty directory named "fugitive:" at the root of my repo. Possibly some unpleasant side-effect of auto-saving a session (via Obsession) when there were fugitive buffers open. Removing this directory fixed the issue.

**Reply by tpope** [reply]
Sounds more like a misbehaving autocommand that creates directories automatically before write.

**Reply by mellery451** [reply]
you are correct - as usual. I will remove the questionable au
